src/polymer-language-server.ts

```diff
@@ -106,12 +106,46 @@ connection.onInitialize((params): InitializeResult => {
   };
 });
 
+async function mapAllInitialDocuments(editorService: EditorService) {

```

Comment:
Name changed in #62 (and it turns out this function isn't needed at all, you get events for each open document)